src/Workspaces/CSharp/Portable/Extensions/TypeSyntaxExtensions.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
```

Comment:
It doesn't look like there is a meaningful change in this file. Consider reverting.